packages/external-integration/project.json

```diff
@@ -18,5 +18,5 @@
             }
         }
     },
-    "tags": ["scope:integration"]
+    "tags": ["scope:shared"]
```

Comment:
why do we need shared scope here?

Reply:
because` integration` scope don't allow to be dependency of another integration (sezzle integration) then external integration as `shared` allow it.